hydroshare/local_settings.cutover

```diff
@@ -189,3 +189,9 @@ HSWS_API_TOKEN = "d1caabf3059ef7ffd99d890aba57100da2e0efc8"
 HSWS_TIMEOUT = 3
 HSWS_PUBLISH_URLS = False
 HSWS_ACTIVATED = False
+
+# HydroShare Data Services
+HS_GEOSERVER = "https://geoserver.hydroshare.org/geoserver"
+HS_GEOSERVER_ESCAPE = {
+    '/': ' '
```

Comment:
These config options look like they belong in settings.py, not local_settings.

Reply:
The config names here are closely related to the HSWS* settings, so it would be good to have the names consistent.

Reply:
867d2f9: Moved these settings to settings.py and changed all name patterns to HSWS*

Reply:
Also, I'm not sure if local_settings.cutover is even used. @dan-palmer1 should investigate

Reply:
Looks like your commit moved all of the HSWS URLs over to settings.py.  I was referring only to the new settings.  This shouldn't be a problem though... it's pointing to a dev geoserver and not activated by default.